src/fontra/views/editor/panel-transformation.js

```diff
@@ -373,6 +492,21 @@ export default class TransformationPanel extends Panel {
           );
       }
     }
+    return staticGlyphControllers;
+  }
+
+  async _transformLayerGlyph(transformation, undoLabel) {
```

Comment:
Btw now that we have `moveObjects()`, I think this method should be renamed `transformSelection()`.

Reply:
Agree. Done.